kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -249,6 +249,22 @@ object KyuubiConf {
       .checkValue(t => t > 0, "must be positive integer")
       .createWithDefault(Duration.ofMinutes(1).toMillis)
 
+  val CREDENTIALS_HADOOP_FS_ENABLED: ConfigEntry[Boolean] =
+    buildConf("credentials.hadoopfs.enabled")
+      .doc("Whether to renew HadoopFS DelegationToken")
+      .version("1.4.0")
+      .booleanConf
+      .createWithDefault(true)
+
+  val CREDENTIALS_HADOOP_FS_URLS: ConfigEntry[Seq[String]] =
+    buildConf("credentials.hadoopfs.urls")
```

Comment:
How about
```
credentials.hadoop.filesystems
credentials.hadoop.enabled

```

Reply:
In reference to the [discussion](https://github.com/apache/spark/pull/16432#issuecomment-270320823) about the serviceName of `HadoopFSCredentialProvider` in Spark, using `hadoopfs` to stand for Hadoop Compatible file systems seems to be more appropriate.

Maybe we, should rename `credentials.hadoopfs.urls` to `credentials.hadoopfs.uris` as they are parsed as java.net.URI objects.